src/main/kotlin/racingCar/controller/RacingController.kt

```diff
@@ -0,0 +1,20 @@
+package racingCar.controller
+
+import racingCar.domain.Racing
+import racingCar.domain.RandomStrategy
+import racingCar.view.InputView
+import racingCar.view.OutputView
+
+fun main() {
+    OutputView.askCarNumber()
+    val carNumber = InputView.number()
+    val racing = Racing(carNumber, RandomStrategy())
+    OutputView.askTryNumber()
+    val tryNumber = InputView.number()
+    OutputView.printResultMsg()
+    (0..tryNumber).forEach {
```

Comment:
이 부분은 코틀린의 `repeat` 함수로 좀 더 간결하게 표현 가능할 거 같아요~
```suggestion
    repeat(tryNumber) {
```
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/repeat.html